clusters/data_node_type.go

```diff
@@ -136,6 +137,9 @@ func (a ClustersAPI) GetSmallestNodeType(r NodeTypeRequest) string {
 		if r.VCPU {
 			return "vcpu-worker"
 		}
+		if r.Fleet {
+			return "m-fleet.xlarge"
```

Comment:
Let's use `md-fleet.xlarge` as that supports photon.
```suggestion
			return "md-fleet.xlarge"
```